src/tests/Reader/NRFReader.t.h

```diff
@@ -31,7 +31,6 @@ TEST_CASE("NRF Reader") {
   REQUIRE(nrf.subfaults[0].tinit == AbsApprox(0.0));
   REQUIRE(nrf.subfaults[0].timestep == AbsApprox(0.0002));
   REQUIRE(nrf.subfaults[0].mu == AbsApprox(0.0));
-  REQUIRE(nrf.source == 1);
```

Comment:
`REQUIRE(nrf.size() == 1);`

Maybe also test that `nrf.subfaults.size()==nrf.centres.size()==nrf.sroffsets.size()` (the syntax used in this comment here is of course not correct)